html-forms/html-forms_Finding-a11y-errors/index.html

```diff
@@ -11,21 +11,35 @@
   </head>
   <body>
     <section>
-      <h1 class="title">Understanding A11y in Forms</h1>
+      <h1 class="title" id="title">Understanding A11y in Forms</h1>
+      <!-- ??? Question to the teacher: is this enough to connect a form 
+          with its headline as following: 
+          <h1 id="title"... <form for="title"... 
```

Comment:
<form for="title"... does not really work here,  <form aria-labelledby="title"... should be used.